source/h1_stream.c

```diff
@@ -32,10 +33,93 @@ static void s_stream_update_window(struct aws_http_stream *stream, size_t increm
     aws_http_connection_update_window(stream->owning_connection, increment_size);
 }
 
+static int s_body_chunks_init(struct aws_h1_stream *stream) {
+    AWS_PRECONDITION(stream);
+    aws_linked_list_init(&stream->body_chunks.chunk_list);
+    stream->body_chunks.current_chunk = NULL;
+    stream->body_chunks.paused = false;
+    return aws_mutex_init(&stream->body_chunks.lock);
+}
+
+void s_clean_up_body_chunk(struct aws_http1_stream_chunk **chunk) {
```

Comment:
static

Reply:
I regex-found a bunch more non-static "s_" functions all over the library and fixed them